logic/overworld.py

```diff
@@ -274,15 +275,15 @@ def __init__(self, options, world_setup, r):
         animal_village.connect(ukuku_prairie, OR(HOOKSHOT, ROOSTER))
         animal_village_connector_left = Location()
         animal_village_connector_right = Location().connect(animal_village_connector_left, PEGASUS_BOOTS)
-        self._addEntrance("prairie_to_animal_connector", ukuku_prairie, animal_village_connector_left, OR(BOMB, BOOMERANG, MAGIC_POWDER, MAGIC_ROD, SWORD)) # passage under river blocked by bush
```

Comment:
Missing closing bracket?

Reply:
Cool, I missed one. Bound to happen eventually, sigh.